README.md

```diff
@@ -111,46 +132,17 @@ Usage: Example  <applianceURL>
 
 `variableId`: The name of an existing variable in Conjur that has a value set and for which the `username` has execute permissions.
 
-`hostFactoryToken`: A host factory token. The easiest way to get a host factory token for testing is to add a hostfactory to a layer using the Conjur CLI command `conjur hostfactory create` and `conjur hostfactory token create`. Take the token returned from that call and pass it as the hostFactoryToken parameter to this example.
+`hostFactoryToken`: A host factory token. The easiest way to get a host factory token for testing is to add a hostfactory to a layer using 
+the Conjur CLI command `conjur hostfactory create` and `conjur hostfactory token create`. Take the token returned from that call 
+and pass it as the hostFactoryToken parameter to this example.
 
-## Example
+## Contributing
 
-```sh
-    // Instantiate a Conjur Client object.
-    //  parameter: URI - conjur appliance URI
-    //  parameter: ACCOUNT - conjur account name
-    //  return: Client object - if URI is incorrect errors thrown when used
-    Uri uri = new Uri("https://myorg.com");
-    Client conjurClient = new Client(uri, account);
-
-    // Login with Conjur credentials like userid and password,
-    // or hostid and api_key, etc
-    //  parameters: username - conjur user or host id for example
-    //              password - conjur user password or host api key for example
-    string conjurAuthToken = conjurClient.Login(username, password);
-
-    // Check if this user has permission to get the value of variableId
-    // That requires execute permissions on the variable
-
-    // Instantiate a Variable object
-    //   parameters: client - contains authentication token and conjur URI
-    //               name - the name of the variable
-    Variable conjurVariable = new Variable(conjurClient, variableId);
-
-    // Check if the current user has "execute" privilege required to get
-    // the value of the variable
-    //   parameters: privilege - string name of the priv to check for
-    bool isAllowed = conjurVariable.Check("execute");
-    if (!isAllowed)
-    {
-        Console.WriteLine("You do not have permissions to get the value of {0}", variableId);
-    }
-    else
-    {
-        Console.WriteLine("{0} has the value: {1}", variableId, conjurVariable.GetValue());
-    }
```

Comment:
This example is still pretty nice (if it works with the new changes) though maybe it should be simplified to just get/set of a variable. I'd keep this and move it to the new location of `Examples` section at the very least though.

Reply:
These examples are all still provided in the example app that comes bundled in the repo

Reply:
`example/Program.cs Line 103`